client/templates/course/toolbar/edit-course.html

```diff
@@ -1,10 +1,15 @@
 <template name="editCourse">
     <ul class="nav nav-pills navbar-right" role="tablist">
         <li>
+        	{{#if owner}}
```

Comment:
Also, this may be included in a separate pull request, as it is not related to profile settings.